config/routes.rb

```diff
@@ -128,10 +132,7 @@
       end
     end
     
-    resource :admin, only: :show do
-      collection do
-        get 'throwException'
-      end
+      get 'admin'
```

Comment:
Do you think this should be something like 'settings' or 'options'? It's a bit confusing to have two admin routes, but it's no big deal.